ansible/roles/olm_setup/tasks/main.yml

```diff
@@ -38,6 +41,23 @@
       src: "{{ olm_repo_dest }}/deploy/okd/manifests/{{ olm_version }}/"
       dest: "{{ _olm_manifests }}"
 
+  - name: "TODO - Temp force OLM namespace until web console is fixed"
+    when: olm_namespace_hack
+    block:
+    - shell: "grep 'openshift-operator-lifecycle-manager' -lr {{ _olm_manifests}}"
+      register: files_to_patch
+    - shell: "sed -i 's|openshift-operator-lifecycle-manager|operator-lifecycle-manager|' {{ item }}"
+      with_items: "{{ files_to_patch.stdout_lines }}"
+
+  - when: olm_namespace == ""
+    block:
+    - name: Get OLM namespace
+      shell: "perl -ne '/name: (.*)$/ && print $1' {{ olm_repo_dest }}/deploy/okd/manifests/{{ olm_version }}/30_00-namespace.yaml"
```

Comment:
Wow, first time I've seen perl run from an ansible playbook... Not sure what to think